examples/java/cluster_example.java

```diff
@@ -0,0 +1,48 @@
+import glide.api.GlideClusterClient;
+import glide.api.logging.Logger;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.GlideClusterClientConfiguration;
+import glide.api.models.configuration.RequestRoutingConfiguration;
+import java.util.concurrent.ExecutionException;
+
+import static glide.api.models.GlideString.gs;
+
+public class Main {
```

Comment:
nit: Please add java doc to the example - explaining what the example is doing.

Reply:
Will do.